lib/base/connection-pool.js

```diff
@@ -133,6 +136,18 @@ class ConnectionPool extends EventEmitter {
           break
         case 'context connection':
           break
+        case 'clientid':
+        case 'client id':
```

Comment:
Can we use just one name (probably `client id`) rather than 2.  I don't think there's value in having variability here. And same with others